packages/eas-cli/src/commandUtils/context/VcsClientContextField.ts

```diff
@@ -0,0 +1,9 @@
+import { getVcsClient } from '../../vcs';
+import { Client } from '../../vcs/vcs';
+import ContextField from './ContextField';
+
+export default class VcsClientContextField extends ContextField<Client> {
+  async getValueAsync(): Promise<Client> {
+    return getVcsClient();
```

Comment:
Ideally, in a follow-up PR, we could get rid of this function so that future command authors aren't tempted to just call this and instead are led to the "pit of success" by only being able to access this via a context. Of course, to do this will require more mocking in tests, so that's why I suggest doing this in a separate PR after this one.

For context (no pun intended), one of the original reasons for having context fields was to prevent command authors from calling getProjectAsync directly without calling a method to set it up. Now, all accesses to the project run the same code that ensures the project is set up correctly.